internal/testplanet/run.go

```diff
@@ -56,6 +56,13 @@ func Run(t *testing.T, config Config, test func(t *testing.T, ctx *testcontext.C
 				}, nil
 			}
 
+			if satelliteDB.Name == "Postgres" {
```

Comment:
You could make this part of the `satellitedbtest.Databases()` result to make it nicer

Reply:
yeah, as I wrote in description its just temporary change. I need to find a better solution.